monalisa.sh

```diff
@@ -0,0 +1,37 @@
+package: MonALISA
+version: "20210818"
+# tag: "1.0.0"
+# source: https://gitlab.cern.ch/jalien/jalien.git
+requires:
+ - JDK
+ - "curl:(?!slc8)"
```

Comment:
Do you really need curl at runtime? Also, given you only need it for building, I would simply drop the dependency.